packages/wordpress-kit/src/lib/tailwindcssPlugin/components/index.ts

```diff
@@ -0,0 +1,6 @@
+import ImageComponent from './Image';
+import PullQuoteComponent from './PullQuote';
+import TableComponent from './Table';
+import Quote from './Quote';
```

Comment:
Is there a good reason to use default exports here instead of named exports?

Reply:
Just personal preference, but if the standard is named exports I can change it